views/templates/front/savedCreditCards.tpl

```diff
@@ -0,0 +1,14 @@
+<!--
+  ~ Altapay module for Prestashop
+  ~
+  ~ Copyright © 2020 Altapay. All rights reserved.
+  ~ For the full copyright and license information, please view the LICENSE
+  ~ file that was distributed with this source code.
+  -->
+
```

Comment:
Do we really want to inject our copyright into the user's site like this?

Reply:
It is basically a signature which is required if we plan to put our plugin on prestashop marketplace.

Reply:
Would it be possible to write it as a smarty comment, then it won't be outputted with the website content.
https://www.smarty.net/docsv2/en/lanuage.basic.syntax.tpl#language.syntax.comments